pom.xml

```diff
@@ -385,6 +391,19 @@
             <version>1.1.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hibernate.javax.persistence</groupId>
+            <artifactId>hibernate-jpa-2.1-api</artifactId>
+            <version>1.0.2.Final</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <!-- Part of Hibernate 6.x, required to run tests against Hibernate 5.x -->
+            <groupId>jakarta.persistence</groupId>
+            <artifactId>jakarta.persistence-api</artifactId>
+            <version>3.1.0</version>
+            <scope>provided</scope>
```

Comment:
If this is required to run the tests then the scope should be `test`?

Reply:
Not sure why, but it's needed for compilation against Hibernate 6, otherwise we got:
```
hazelcast-hibernate/hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/AbstractHazelcastCacheRegionFactory.java:[47,17] cannot access jakarta.persistence.PersistenceException
[ERROR]   class file for jakarta.persistence.PersistenceException not found
```

Reply:
With the latest changes this seems to work fine with `test` scope.

Reply:
it fails with `test` scope: `mvn clean verify -Dhazelcast.version=5.1.3 -Dhibernate.core.version=6.1.3.Final`